docs/docs/how_to/document_loader_sql_database.ipynb

```diff
@@ -0,0 +1,360 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# SQL Database\n",
+    "\n",
+    "## About\n",
+    "\n",
+    "The `SQLDatabaseLoader` loads records from any database supported by\n",
+    "[SQLAlchemy], see [SQLAlchemy dialects] for the whole list of supported\n",
+    "SQL databases and dialects.\n",
+    "\n",
+    "For talking to the database, the document loader uses the [SQLDatabase]\n",
+    "utility from the LangChain integration toolkit.\n",
+    "\n",
+    "You can either use plain SQL for querying, or use an SQLAlchemy `Select`\n",
+    "statement object, if you are using SQLAlchemy-Core or -ORM.\n",
+    "\n",
+    "You can select which columns to place into the document, which columns\n",
+    "to place into its metadata, which columns to use as a `source` attribute\n",
+    "in metadata, and whether to include the result row number and/or the SQL\n",
+    "query expression into the metadata.\n",
+    "\n",
+    "## What's inside\n",
+    "\n",
+    "This notebook covers how to load documents from an [SQLite] database,\n",
+    "using the [SQLAlchemy] document loader.\n",
+    "\n",
+    "It loads the result of a database query with one document per row.\n",
+    "\n",
+    "[SQLAlchemy]: https://www.sqlalchemy.org/\n",
+    "[SQLAlchemy dialects]: https://docs.sqlalchemy.org/en/latest/dialects/\n",
+    "[SQLDatabase]: https://python.langchain.com/docs/integrations/toolkits/sql_database\n",
```

Comment:
Linking to the [SQLDatabase Toolkit](https://python.langchain.com/docs/integrations/tools/sql_database/) was probably wrong?

```suggestion
    "[SQLDatabase]: https://python.langchain.com/api_reference/community/utilities/langchain_community.utilities.sql_database.SQLDatabase.html\n",
```